lib/ansible/modules/source_control/gitlab/gitlab_project_variable.py

```diff
@@ -69,6 +72,19 @@
       ACCESS_KEY_ID: abc123
       SECRET_ACCESS_KEY: 321cba
 
+- name: Set or update some CI/CD variables
+  gitlab_project_variable:
+    api_url: https://gitlab.com
+    api_token: secret_access_token
+    project: markuman/dotfiles
+    purge: false
+    vars:
+      ACCESS_KEY_ID: abc123
+      SECRET_ACCESS_KEY:
+        value: 3214cbad
+        masked: True
```

Comment:
```suggestion
        masked: true
```
(Same for next line)